Improvements to the Step by step tutorial (#6622)
Particular/docs.particular.net: tutorials/nservicebus-step-by-step/1-getting-started/tutorial.md

When setting up the endpoint configuration you can choose how you want to serialize your messages. For this tutorial, we will be configuring the endpoint to use `SystemJsonSerializer` which uses the .NET `System.Text.Json` serializer.

#### Transport

snippet: LearningTransport

This setting defines the [**transport**](/transports/) that NServiceBus will use to send and receive messages. We are using the [Learning transport](/transports/learning/), which is bundled in the NServiceBus.Core library as a starter transport for learning how to use NServiceBus without additional dependencies. All other transports are provided using different NuGet packages.

Capturing the `transport` settings in a variable as shown will make things easier in [a later lesson](../3-multiple-endpoints/) when we start defining message routing rules.

### Starting up

> At the end of the `Main` method, after the configuration code, add the following lines which will: start the endpoint, keep it running until you press the <kbd>Enter</kbd> key, that will shut it down.

Reviewer: ```suggestion
At the end of the `Main` method, after the configuration code, add the following lines which will start the endpoint and keep it running until you press the <kbd>Enter</kbd> key to shut it down.
```

snippet: Startup

> The endpoint is initialized according to the settings defined by the `EndpointConfiguration` class. Once the endpoint starts, any changes made to the configuration information won't be applied.
> In order to apply configuration changes, you will need to restart the endpoint.

Reviewer: ```suggestion
The endpoint is initialized according to the settings defined by the `EndpointConfiguration` class. Once the endpoint starts, any changes made to the configuration won't be applied until you restart the endpoint.
```

When you run the endpoint for the first time, the endpoint will:

 * Display its logging information, which is [written to a file, Trace, and the Console](/nservicebus/logging/#default-logging). NServiceBus also logs to multiple levels, so you can [change the log level](/nservicebus/logging/#default-logging-changing-the-defaults-changing-the-logging-level) from `INFO` to `DEBUG` in order to get more information.
 * Display the [status of your license](/nservicebus/licensing/).
 * Attempt to add the current user to the "Performance Monitor Users" group so that it can write [performance counters](/monitoring/metrics/performance-counters.md) to track its health and progress.

>  * Create file-based "queues" in the `.learningtransport` directory inside your solution directory. This is why we recommend to add `.learningtransport` to your source control system's ignore file.

Reviewer: ```suggestion
 * Create file-based "queues" in the `.learningtransport` directory inside your solution directory. We recommend adding `.learningtransport` to your source control system's ignore file.
```

or 

```suggestion
 * Create file-based "queues" in the `.learningtransport` directory inside your solution directory. Add `.learningtransport` to your source control system's ignore file.
```
